Sources/AWSLambdaRuntimeCore/HTTPClient.swift

```diff
@@ -26,6 +26,7 @@ internal final class HTTPClient {
 
     private var state = State.disconnected
     private var executing = false
+    private let headers = HTTPHeaders([("User-Agent", "Swift-Lambda/Unknown")])
```

Comment:
I'd do this one `static` so we can save a few allocations (and a little bit of space). Given that we're editing this line anyway, I'd also make the `user-agent` lower-case (just to be more compatible with HTTP/2 if Lambda were to ever switch to H2).